.github/workflows/test-invoke-pip.yml

```diff
@@ -85,6 +85,14 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: pip
           cache-dependency-path: pyproject.toml
+      
+      - name: install frontend deps
+        run: yarn install --frozen-lockfile
+        working-directory: invokeai/frontend
+      
+      - name: build frontend
+        run: yarn build
+        working-directory: invokeai/frontend
```

Comment:
not sure how this helps the CLI-based tests, while it would break the tests if there is a problem which would prevent building the frontend.

Reply:
Frontend builds probably need to be tested as well if a PR needs to be checked IMO, and also it's because the code imports the frontend folder path at a top level import that also gets included when calling the CLI